interface/controller/student_controller.go

```diff
@@ -0,0 +1,58 @@
+package controller
+
+import (
+	"encoding/csv"
```

Comment:
By convention, the imports should be ordered as follows:
```
core 

local

third partys
```

Reply:
Done